hugo/data/home/home.yml

```diff
@@ -31,6 +29,9 @@ projects:
 
 roadmap:
   display: true
+  content:
```

Comment:
Since the signature is already planned, it could be prepared in advance. Then in the markup, if there is no signature url configured, it will be used the default text